.gitignore

```diff
@@ -1,5 +1,5 @@
 dist
-*classes
+*.class
```

Comment:
@Gitomain  We can add `*.class` but we need to keep `classes` as some folders are named like this. Can you update the PR? Otherwise I can fix during merging.